Source/Atomic/UI/UISelectItem.h

```diff
@@ -80,6 +80,8 @@ class UISelectItemSource : public Object
 
     void Clear() { items_.Clear(); }
 
+    const String& GetItemStr(int index);
+
```

Comment:
I've been trying to remember to add /// comment blocks above methods when adding them